osv/models.py

```diff
@@ -259,6 +259,8 @@ class Bug(ndb.Model):
   aliases: list[str] = ndb.StringProperty(repeated=True)
   # Related IDs.
   related: list[str] = ndb.StringProperty(repeated=True)
+  # Upstream IDs.
+  upstream: list[str] = ndb.StringProperty(repeated=True)
```

Comment:
Please rename this to upstream_raw as described in the doc.

Reply:
Can you also expand the docstring above this to explain that this is upstream ids directly from the import and does not include any transitive upstreams.